libs/application-generic/src/schemas/control/layout-control.schema.ts

```diff
@@ -4,10 +4,11 @@ import { UiComponentEnum, UiSchema, UiSchemaGroupEnum } from '@novu/shared';
 import { JSONSchemaEntity } from '@novu/dal';
 import { defaultOptions } from './shared';
 
+// email layout schema is a subset of the email control schema
 const layoutZodSchema = z.object({
   email: z
     .object({
-      content: z.string().min(1),
+      body: z.string().min(1),
```

Comment:
updated the `content` to `body` as the layout email controls are subset of email step controls